homeassistant/components/squeezebox/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Platform for sensor integration for squeezebox."""
+
+from __future__ import annotations
+
+import logging
+from typing import cast
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import EntityCategory, UnitOfTime
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import SqueezeboxConfigEntry
+from .const import (
+    STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+    STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+    STATUS_SENSOR_INFO_TOTAL_DURATION,
+    STATUS_SENSOR_INFO_TOTAL_GENRES,
+    STATUS_SENSOR_INFO_TOTAL_SONGS,
+    STATUS_SENSOR_LASTSCAN,
+    STATUS_SENSOR_NEWPLUGINS,
+    STATUS_SENSOR_NEWVERSION,
+    STATUS_SENSOR_OTHER_PLAYER_COUNT,
+    STATUS_SENSOR_PLAYER_COUNT,
+)
+from .entity import LMSStatusEntity
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:album",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:account-music",
+    ),
```

Comment:
Unit of measurement?

Reply:
Its a total count so I wouldn't think so

Reply:
It does tell the amount of albums and artists, I think it makes sense to add a unit of measurement

Reply:
Indeed a we have that in the sensor name that is nicely translatable, I would argue that he unit is a count.  It does seem to be a bit redundant and the tenths of the unit will me wrong for 1 unit also ;-), but if you want it I will do it.